main-app/app/scripts/providers/game-model.js

```diff
@@ -0,0 +1,126 @@
+(function () {
+    'use strict';
+
+    angular.module('Tombola.NoughtsAndCrosses.Providers')
+        .factory('GameModel',['$http', '$timeout', '$location', 'Proxy', 'Audio', function($http, $timeout, $location, proxy, audio) {
+            var Model = function() {
+                var me = this;
+
+                me.currentGrid = [[{key: "0", value: "0"}, {key: "1", value: "0"}, {key: "2", value: "0"}],
```

Comment:
Not sure what this array of object is buying you as the "key" is just a placebo array index.